docs/content/02-tutorials/07-cost-tab.md

```diff
@@ -0,0 +1,56 @@
+
++++
+title = "f. Cost Tab 📈"
+weight = 27
++++
+
+This tutorial tells you how to view cluster spending costs and created budgets for clusters
+
+## Setup
+
+1. Cost Allocation Tags allow us to track spending of parallelclusters from parallelcluster specific tags, so first we must activate those tags.
```

Comment:
Let's make this the introduction, not a numbered step.

Reply:
Resolved